tests/test_roles.py

```diff
@@ -53,11 +53,15 @@ def test_role_statement(self):
                 role.set_policy("Something else")
             self.assertEqual(role.get_policy(), statement)
 
+        statement = create_test_statements(150)
         with self.subTest("an error is returned when a policy exceed 10 Kb"):
```

Comment:
Would suggest either labeling these subTests with distinct messages, or putting the `set_policy()` and `Role.create()` calls in the same subtest.

Reply:
fixed names